libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/gutenberg/GutenbergPropsBuilder.kt

```diff
@@ -14,7 +14,7 @@ data class GutenbergPropsBuilder(
     private val enableXPosts: Boolean,
     private val enableUnsupportedBlockEditor: Boolean,
     private val unsupportedBlockEditorSwitch: Boolean,
-    private val enableAudioBlock: Boolean,
+    private val isAudioBlockMediaUploadEnabled: Boolean,
```

Comment:
These changes come from this [PR](https://github.com/wordpress-mobile/WordPress-Android/pull/14680), that it was required to merge it here due to conflicts in the Gutenberg reference.